component/core/src/main/java/org/exoplatform/social/core/jpa/search/ProfileIndexingServiceConnector.java

```diff
@@ -225,6 +226,9 @@ private Document getDocument(String id) {
     fields.put("avatarUrl", profile.getAvatarUrl());
     fields.put("userName", identity.getRemoteId());
     fields.put("email", profile.getEmail());
+    if (isExternal) {
```

Comment:
external property should be reindexed when its value is changed

Reply:
+1
`      fields.put("external", String.valueOf(isExternal));`
To delete `if (isExternal) `

Reply:
Updated!